package.json

```diff
@@ -28,12 +28,12 @@
   "dependencies": {
     "atom-package-deps": "7.2.3",
     "consistent-path": "2.0.3",
-    "crypto-random-string": "3.3.1",
+    "crypto-random-string": "4.0.0",
```

Comment:
We should not switch to 4.0.0 ever. They have switched to pure ESM.
https://github.com/sindresorhus/crypto-random-string/releases

They have also archived the repository to prevent people from creating issues. 
https://github.com/sindresorhus/crypto-random-string